l10cloud-formation/lesson10security.yml

```
@@ -0,0 +1,74 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources: 
+#ec2のセキュリティグループ
+  lesson10ec2security:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupName: lesson10ec2security
+      GroupDescription: Security group for SSH HTTP access
+      VpcId: !ImportValue lesson10vpcoutName
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 121.105.112.83/32
```

インバウンドのソースIPは恐らくご自身のインターネットパブリックIPかなと理解しましたので、ここは固定値にしないほうが良いかなと思います！
```suggestion
        - IpProtocol: tcp
          FromPort: 22
          ToPort: 22
          CidrIp: !Ref CiderIpFromInernet
```

---

Statusの項目を作り、参照させるようにしました